source/effect_module.hpp

```diff
@@ -193,8 +193,8 @@ namespace reshadefx
 		texture_address_mode address_u = texture_address_mode::clamp;
 		texture_address_mode address_v = texture_address_mode::clamp;
 		texture_address_mode address_w = texture_address_mode::clamp;
-		float min_lod = -3.402823466e+38f; // FLT_MAX
-		float max_lod = +3.402823466e+38f;
+		float min_lod = std::numeric_limits<float>::lowest();
+		float max_lod = std::numeric_limits<float>::max();
```

Comment:
This cannot be used here, since the `<limit>` header is not included at this point (and it seems unnecessary to include it just for this). This compiles with MSVC, because the STL in MSVC includes `<limit>` in `<vector>`, but it does not compile on Clang/GCC. All the `effect_*` files need to retain Linux compatibility (for VkBasalt and others), so this would break that.